com.wiz.Note.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "com.wiz.Note",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.12",
+  "sdk": "org.kde.Sdk",
+  "separate-locales": false,
+  "command": "WizNote",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--persist=.wiznote", 
+    "--persist=.local",
+    "--filesystem=xdg-documents"
+  ],
+  
+  "modules": [
+    {
+      "name": "WizNote",
+      "buildsystem": "cmake",
+      "builddir": true,
+      "post-install": [
+        "rm -rf /app/share/icons/hicolor/{8x8,16x16,24x24,32x32,48x48,512x512}",
+        "mv /app/share/icons/hicolor/64x64/apps/{wiznote.png,com.wiz.Note.png}",
+        "mv /app/share/icons/hicolor/128x128/apps/{wiznote.png,com.wiz.Note.png}",
+        "mv /app/share/icons/hicolor/256x256/apps/{wiznote.png,com.wiz.Note.png}",
+        
+        "rm /app/share/applications/wiznote.desktop",
+        "install -D -m0644 -t /app/share/applications/ com.wiz.Note.desktop",
```

Comment:
Why not to use `rename-desktop-file` instead?

Reply:
I am using a custom desktop and I don't know which way is better.
The docs for rename-desktop-file and rename-icon is not detailed enough and I don't know if these commands will just rename the files or copy the files to corresponding dir after rename.

Reply:
It will rename them, rename some of the fields in the file and copy it to the correct place (so long as the file it was copying from was in the right place to begin with)

Reply:
Thanks, I will update the manifest.